website/docs/installation/documentdb/deb.md

```diff
@@ -54,3 +56,31 @@ CREATE EXTENSION documentdb CASCADE;
 ```
 
 You can now go ahead and set up FerretDB by following [this installation guide](../ferretdb/deb.md).
+
+## Updating to a new DocumentDB release
```

Comment:
There is no need to mention DocumentDB in the section header, since the page is all about DocumentDB